meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -54,17 +54,21 @@ do_install:append:lmp() {
     install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}${nonarch_libdir}/tmpfiles.d/aktualizr-lite.conf
 }
 
-PACKAGES += "${PN}-get ${PN}-lite ${PN}-lite-lib ${PN}-lite-dev"
+PACKAGES += "${PN}-apps ${PN}-get ${PN}-lite ${PN}-lite-lib ${PN}-lite-dev ${PN}-offline "
+FILES:${PN}-apps = " \
```

Comment:
Before it was part of aktualizr-lite, and now it is getting moved to aktualizr-apps, while it should be aktualizr-lite-apps, right?

Same for offline.